hr_timesheet_overtime/models/resource_overtime.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+# Copyright 2019 Coop IT Easy SCRLfs
+#   - Vincent Van Rossem <vincent@coopiteasy.be>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+import datetime
+
+from openerp import models, api, fields
+
+
+class ResourceOvertime(models.Model):
+    _name = "resource.overtime"
+    _description = "Resource Overtime"
+
+    name = fields.Char(required=True)
+    company_id = fields.Many2one(
+        "res.company",
+        string="Company",
+        default=lambda self: self.env["res.company"]._company_default_get(),
+    )
+    rate_ids = fields.One2many(
+        "resource.overtime.rate",
+        "overtime_id",
+        string="Overtime Rate",
+        copy=True,
+    )
```

Comment:
I like to specify the named parameters in xTomany fields:

```py
    rate_ids = fields.One2many(
        comodel_name="resource.overtime.rate",
        inverse_name="overtime_id",
        string="Overtime Rate",
        copy=True,
    )
```

Reply:
done